scalding-beam/src/test/scala/com/twitter/scalding/beam_backend/BeamBackendTests.scala

```diff
@@ -389,12 +389,55 @@ class BeamBackendTests extends FunSuite with BeforeAndAfter {
     )
   }
 
+  test("Force to Disk execution") {
+    val bmode = BeamMode.default(pipelineOptions)
+    val tmpPath1 = tmpPath("tp1")
+    val tmpPath2 = tmpPath("tp2")
+
+    val tmpDir = new File(tmpPath("forced"))
+    tmpDir.mkdirs()
+
+    val forcedExecution =
+      TypedPipe
+        .from(Seq(5, 3, 2, 6, 1, 4))
+        .map { e =>
+          // This tests that forcedExecution runs only once. If it runs twice createFile will return false
```

Comment:
did you verify that this test will fail if you remove the code that prevents rerunning?

A wise man once told me: you haven't seen a test pass until you have seen it fail.

Reply:
Haha good one.
Yeah I tested it without forceToDisk and it failed. I can add a test which fails.